pages/index.js

```diff
@@ -88,10 +80,22 @@ export default function HomePage() {
         height={200}
       />
       <LinkContainer>
-        <StyledLink href="/profile/">Profile</StyledLink>
-        <StyledLink href={`/events/upcoming`}>Upcoming Appointments</StyledLink>
-        <StyledLink href={`/events/latest`}>Latest Appointments</StyledLink>
-        <StyledLink href={`/events/wishlist`}>Your Wishlist</StyledLink>
+        <StyledLink href={`/events/latest`}>
+          <SVGIcon variant="last" width="4rem" />
+          <StyledParagraph>latest</StyledParagraph>
+        </StyledLink>
+        <StyledLink href={`/events/upcoming`}>
+          <SVGIcon variant="next" width="4rem" />
+          <StyledParagraph>Next</StyledParagraph>
+        </StyledLink>
+        <StyledLink href={`/events/wishlist`}>
+          <SVGIcon variant="wish" width="4rem" />
+          <StyledParagraph>Wishes</StyledParagraph>
+        </StyledLink>
+        <StyledLink href={`/map`}>
```

Comment:
you don't need the brackets and the template literals:

```suggestion
        <StyledLink href="/map">
```

Reply:
Done